deployment/src/test/resources/com/github/mcollovati/quarkus/hilla/deployment/crud/panache/application.properties

```diff
@@ -3,4 +3,6 @@ quarkus.datasource.jdbc.url=jdbc:h2:mem:sample
 quarkus.datasource.jdbc.max-size=8
 quarkus.hibernate-orm.database.generation=drop-and-create
 quarkus.hibernate-orm.log.sql=true
-quarkus.hibernate-orm.sql-load-script=import.sql
\ No newline at end of file
+quarkus.hibernate-orm.sql-load-script=import.sql
+
+quarkus.package.filter-optional-dependencies=true
```

Comment:
What is changing this?

Reply:
This was a quick test. It has been pushed by mistake

Reply:
removed